### PR TITLE
Fix namespace

### PR DESCRIPTION
**Pull request by chihiro-adachi**

### CODE REVIEW

File: docs/en/reference/second-level-cache.rst [hunk 1]
```diff
@@ -153,21 +153,21 @@ Built-in cached persisters
...
+    | READ_WRITE            | Doctrine\\ORM\\Cache\\Persister\\Entity\\ReadWriteCachedEntityPersister                   |
+    +-----------------------+-------------------------------------------------------------------------------------------+
+    | NONSTRICT_READ_WRITE  | Doctrine\\ORM\\Cache\\Persister\\Entity\\NonStrictReadWriteCachedEntityPersister          |
+    +-----------------------+-------------------------------------------------------------------------------------------+
+    | READ_ONLY             | Doctrine\\ORM\\Cache\\Persister\\Collection\\ReadOnlyCachedCollectionPersister            |
+    +-----------------------+-------------------------------------------------------------------------------------------+
+    | READ_WRITE            | Doctrine\\ORM\\Cache\\Persister\\Collection\\ReadWriteCachedCollectionPersister           |
+    +-----------------------+-------------------------------------------------------------------------------------------+
+    | NONSTRICT_READ_WRITE  | Doctrine\\ORM\\Cache\\Persister\\Collection\\NonStrictReadWriteCacheCollectionPersister   |
```
Comment (phansys) [inline docs/en/reference/second-level-cache.rst]: Did you mean `Doctrine\ORM\Cache\Persister\Collection\NonStrictReadWriteCachedCollectionPersister`?

**Review by phansys — CHANGES_REQUESTED** [review]
Status: Needs Work.

File: docs/en/reference/second-level-cache.rst (hunk 1 above)
Comment (chihiro-adachi) [inline docs/en/reference/second-level-cache.rst]: sorry, `d` was missing.
NonStrictReadWriteCache`d`CollectionPersister is correct.

fixed https://github.com/doctrine/doctrine2/pull/6087/commits/4a4491b7e491ad2c2273e3723d3eeb745c85767e

**Comment by Ocramius** [comment]
@chihiro-adachi thanks!

**Pull request merged**